atomistics/calculators/ase.py

```diff
@@ -120,16 +118,15 @@ def _calc_md_step_with_ase(
 ):
     structure.calc = ase_calculator
     MaxwellBoltzmannDistribution(atoms=structure, temperature_K=temperature)
-    ASEOutputMolecularDynamics = OutputMolecularDynamics(
-        **{k: getattr(ASEExecutor, k) for k in OutputMolecularDynamics.keys()}
-    )
     cache = {q: [] for q in output_keys}
     for i in range(int(run / thermo)):
         dyn.run(thermo)
-        calc_dict = ASEOutputMolecularDynamics.get(
-            ASEExecutor(ase_structure=structure, ase_calculator=ase_calculator),
-            *output_keys,
+        ase_instance = ASEExecutor(
+            ase_structure=structure, ase_calculator=ase_calculator
         )
+        calc_dict = OutputMolecularDynamics(
+            **{k: getattr(ase_instance, k) for k in OutputMolecularDynamics.keys()}
+        ).get(*output_keys)
```

Comment:
But this change requires changing the other calls as well. The important part is the change in output from:
```
    def get(self, engine, *output: str) -> dict:
        return {q: getattr(self, q)(engine) for q in output}
```
To:
```
    def get(self, *output: str) -> dict:
        return {q: getattr(self, q)() for q in output}
```
By no longer providing the `engine` as parameter we can simplify the code as illustrated by the ASE example above. But this affects all output classes at the same time, so all output classes need to be updated.

Reply:
Ahh, ok, I had totally failed to grok the change in `Output`, I apologize. A version of the final sentence there would be perfect as the PR description.

Ok, then I am not sold on introducing variables to hold the `XProperties` rather than using them in-line, but I also don't object to it. And I like the simplification of the `Output` initialization/`get`, that's a nice change.